examples/README.md

```diff
@@ -34,14 +34,24 @@ The kitchen-sink example application is hosted on Vercel, including preview depl
 
 #### Configuring Auth0
 
-Go to the Application Settings on your [Auth0 dashboard](https://manage.auth0.com/) and make sure to configure the following:
+As every environment in Vercel, including preview deployments, has its unique URL, your Auth0 application needs to be configured to allow the corresponding Callback and Logout URLs.
+This can be done manually, by going to the Application Settings on your [Auth0 dashboard](https://manage.auth0.com/) and make sure to configure the following:
 
 | Setting               | Description                                                                                                                                                            |
 | --------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| Allowed Callback URLs | Should be set to `https://{YOUR_VERCEL_PROJECT_NAME}-*.vercel.app/api/auth/callback` when deploying to vercel. |
-| Allowed Logout URLs   | Should be set to `https://{YOUR_VERCEL_PROJECT_NAME}-*.vercel.app/` when deploying to vercel.   
+| Allowed Callback URLs | Should be set to `https://{YOUR_VERCEL_URL}.vercel.app/api/auth/callback` when deploying to vercel. |
+| Allowed Logout URLs   | Should be set to `https://{YOUR_VERCEL_URL}.vercel.app/` when deploying to vercel.   
 
-**Note**: As we are making use of Preview Deployments, we need to configure the above URLs using wildcards, as every preview deployment will get a unique URL.
+##### Automating
```

Comment:
Let's not include this section for now since it's an optional step and the instructions are already quite long

Reply:
I removed it.